protocol/v2/ssv/runner/committee.go

```diff
@@ -426,9 +431,13 @@ func (cr *CommitteeRunner) ProcessPostConsensus(ctx context.Context, logger *zap
 				syncCommitteeMessagesToSubmit[validator] = syncMsg
 
 			} else if role == spectypes.BNRoleAttester {
-				att := sszObject.(*phase0.Attestation)
+				att := sszObject.(*spec.VersionedAttestation)
 				// Insert signature
-				att.Signature = specSig
+				att, err = specssv.VersionedAttestationWithSignature(att, specSig)
+				if err != nil {
+					anyErr = errors.Wrap(err, "could not insert signature in versioned attestation")
```

Comment:
can we not use `errors.Wrap` for new code? if we need it, can we have a comment on why?